flipkart_scraper.py

```diff
@@ -0,0 +1,120 @@
+# The front end can make use of the passing parameters
+# In the dropdown list, the developer can give the Words like Laptops, Mobiles, Cameras, Headphones, Air conditioners etc
+# Also the user can define the limit of number of pages to scrap [It is used in both functions]
+def get_url(dropdownselection,limit):
```

Comment:
It'd be better if the Python variables were in camel_case. Like, `dropdown_selection`